.github/workflows/release.yml

```diff
@@ -46,13 +46,27 @@ jobs:
         git config --local user.name "${GITHUB_ACTOR}"
 
     - name: Build & Publish
+      id: build
       run: |
         echo Version level: ${{ github.event.inputs.versionLevel }}
         npm ci
         npm run build --if-present
-        npm version ${{ github.event.inputs.versionLevel }} --message "${{ github.event.inputs.commitMessage }}"
+        VERSION=$(npm version ${{ github.event.inputs.versionLevel }} --message "${{ github.event.inputs.commitMessage }}")
+        VERSION=${VERSION:1} # remove 'v' prefix
+        echo "##[set-output name=version;]$VERSION"
         git push --follow-tags
         npm config set //registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN
         npm publish --access public
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+    - name: Publish Docker image to Docker Hub
+      env:
+        DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      run: |
+        REPO=cyclonedx/cyclonedx-node
+        VERSION=${{ steps.build.outputs.version }}
+        docker login --username coderpatros --password "$DOCKER_TOKEN"
```

Comment:
`coderpatros` needs to be changed to `$DOCKERHUB_USERNAME `
`$DOCKER_TOKEN` needs to be changed to `$DOCKERHUB_TOKEN`.

Both DOCKERHUB_USERNAME and DOCKERHUB_TOKEN are secrets defined at the GitHub organization level. I assume DOCKER_TOKEN is likely unique to this repo and should be deleted in favor of the organizational secrets.

Reply:
Done! I think that for now this repo doesn't yet publish to docker so no variables was defined.